Makefile

```diff
@@ -83,49 +83,47 @@ docker-images:
 # Rules for verification, formatting, linting, testing and cleaning #
 #####################################################################
 
-.PHONY: revendor
-revendor:
+.PHONY: tidy
+tidy:
 	@GO111MODULE=on go mod tidy
-	@GO111MODULE=on go mod vendor
-	@chmod +x $(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/*
-	@chmod +x $(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/.ci/*
-	@$(REPO_ROOT)/hack/update-github-templates.sh
-	@ln -sf ../vendor/github.com/gardener/gardener/hack/cherry-pick-pull.sh $(HACK_DIR)/cherry-pick-pull.sh
+	@mkdir -p $(REPO_ROOT)/.ci/hack && cp $(GARDENER_HACK_DIR)/.ci/* $(REPO_ROOT)/.ci/hack/ && chmod +xw $(REPO_ROOT)/.ci/hack/*
```

Comment:
Do we not need to copy `$(GARDENER_HACK_DIR)/generate-controller-registration.sh` as well similar to https://github.com/gardener/gardener-extension-shoot-networking-filter/pull/111?

Reply:
It's at least not added to the AWS extension ([ref](https://github.com/gardener/gardener-extension-provider-aws/blob/d75666468dda0fe76ff131d91ca24921dccfa578/Makefile#L106)) and I don't see a reason why this was done. Maybe @MartinWeindel can tell?

Reply:
Checked offline with @MartinWeindel. From his point of view, this should work. Hence, I am fine with it.